src/gateway/router/routes/contractSourceRoute.ts

```diff
@@ -0,0 +1,60 @@
+import Router from '@koa/router';
+import { Benchmark } from 'warp-contracts';
+
+export async function contractSourceRoute(ctx: Router.RouterContext) {
```

Comment:
we will need to configure a cache for this new endpoint - what TTL do you think is reasonable here?